config.yaml

```diff
@@ -14,7 +14,7 @@ options:
   external_hostname:
     type: string
     description: "External hostname this gunicorn app should respond to (required)."
```

Comment:
Can we update the description to say if left blank the deployed juju application name will be used?